### ISSUE TITLE
Clean generated class

### ISSUE THREAD

**Issue by nirodg** [nirodg/easyrs#29]
- [x] Use simple name instead of canonical one for the DTO
- [x] Possibility to add empty line inside the methods 
- [x] Add `@Generated` before the class

It should look like
```java
package com.dorinbrage.github.jme.note.rest;

// Here the imports

@SuppressWarnings("unchecked")
@Generated(
      value = "com.dbrage.lib.easyrs.processor", 
      date = "2017-02-20T18:16:47+0100",
      comments = "version: 1.0.0.CR2, compiler: javac, environment: Java 1.8.0_65 (Oracle Corporation)"
)
public class NoteRestTestEndpoint extends Container<NoteDto, NoteRest> {

  // Here you can define your global variables

  @Before
  public void setUp() {
    // Here you can initialize your variables

  }

  @Test
  public void getAll() {
    List<NoteDto> entities = (ArrayList<NoteDto>) getData(GET_ALL);
    List<NoteDto> fetchedEntities = (ArrayList<NoteDto>) getClient().getAll();
    Assert.assertEquals(entities.size(), fetchedEntities.size());
  }

  @Test
  public void create() {
    NoteDto entity = (NoteDto) getData(PUT);
    Assert.assertNotNull(entity);

    NoteDto fetchedEntity = (NoteDto) getClient().put(entity);
    Assert.assertNotNull(fetchedEntity);
    Assert.assertEquals(entity, fetchedEntity);

  }

  @Test
  public void update() {
    NoteDto entity = (NoteDto) getData(POST);
    Assert.assertNotNull(entity);

    entity = (NoteDto) getClient().put(entity);
    Assert.assertNotNull(entity);

    NoteDto fetchedEntity = (NoteDto) getClient().post(entity.getGuid(), entity);
    Assert.assertNotNull(fetchedEntity);
    Assert.assertEquals(entity, fetchedEntity);
  }

  @Test
  public void delete() {
    NoteDto entity = (NoteDto) getData(DELETE);
    Assert.assertNotNull(entity);

    entity = (NoteDto) getClient().put(entity);
    Assert.assertNotNull(entity);

    NoteDto fetchedEntity = (NoteDto) getClient().delete(entity.getGuid());
    Assert.assertNull(fetchedEntity);
  }

}

```

**Comment by nirodg** [comment]
Fix committed  with cdf28f8 and it can be tested

**Comment by nirodg** [comment]
Working as expected therefore issue closed
